woocommerce/wc-admin
# Remove test code/examples and disable the inbox activity panel.

This should be easy to review since it's 99% 🔴changes 😄. Fixes #341, #340, #339.

This PR removes some of the test/examples we've accumulated, now that we have charts and summary numbers on other pages. It also removes the inbox tab since we are a ways off from being able to support it.

Note: It seems like the positioning on the activity panel tabs is a little off. I'm guessing maybe a regression from the IE11 fix? cc @Aljullu. I didn't fix it here because I'm out of the house today and don't have my Windows machine on me to make sure I don't rebreak it there -- but I'll fix it later if you don't get to it first.

To Test:
* Check in the activity panel and verify the inbox tab is gone. Shrink your screen down to mobile and open the activity panel. Make sure it defaults to orders.
* Check the dashboard and verify test elements are gone.
* Check the analytics navigation and make sure the test report is gone.
